src/20240707_AaveV3Ethereum_IncreaseCCIPFacilitatorCapacity/AaveV3Ethereum_IncreaseCCIPFacilitatorCapacity_20240707.t.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3Ethereum} from 'aave-address-book/AaveV3Ethereum.sol';
+
+import {ProtocolV3TestBase} from 'aave-helpers/ProtocolV3TestBase.sol';
+import {AaveV3Ethereum_IncreaseCCIPFacilitatorCapacity_20240707} from './AaveV3Ethereum_IncreaseCCIPFacilitatorCapacity_20240707.sol';
+
+interface UpgradeableLockReleaseTokenPool {
+  function getBridgeLimit() external returns (uint256);
+
+  function setBridgeLimit(uint256 limit) external;
+}
+
+/**
+ * @dev Test for AaveV3Ethereum_IncreaseCCIPFacilitatorCapacity_20240707
+ * command: FOUNDRY_PROFILE=mainnet forge test --match-path=src/20240707_AaveV3Ethereum_IncreaseCCIPFacilitatorCapacity/AaveV3Ethereum_IncreaseCCIPFacilitatorCapacity_20240707.t.sol -vv
+ */
+contract AaveV3Ethereum_IncreaseCCIPFacilitatorCapacity_20240707_Test is ProtocolV3TestBase {
+  AaveV3Ethereum_IncreaseCCIPFacilitatorCapacity_20240707 internal proposal;
+
+  function setUp() public {
+    vm.createSelectFork(vm.rpcUrl('mainnet'), 20257789);
+    proposal = new AaveV3Ethereum_IncreaseCCIPFacilitatorCapacity_20240707();
+  }
+
```

Comment:
Similar to this [test](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20240528_Multi_GHOCrossChainLaunch/AaveV3E2E_GHOCrossChainLaunch_20240528.t.sol#L241), I would add an e2e bridge test where before proposal execution we expect the the test to fail when facilitator is full, and after the limit increase we can bridge fine.